subprojects/fossil-test.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = fossil-test-1.2.8
+source_url = https://github.com/fossillogic/fossil-test/archive/refs/tags/v1.2.8.tar.gz
+source_filename = fossil-test-1.2.8.tar.gz
+source_hash = 4ce943bc7395a3a5287b1cc68c12802be360d109506238c1657dc7fba1a98a92
+
+[provide]
+fossil-test = fossi_test_dep
```

Comment:
Should this be `fossil_test_dep`?

Since upstream requires a very new Meson, it'd be nice if the upstream `meson.build` had `meson.override_dependency('fossil-test', fossil_test_dep)` so the wrap file could just say `dependency_names = fossil-test` and avoid the hardcoded variable name.  (And respectively for `pizza-test`.)

Reply:
I'll be sure to use it for the next release, not sure it will point to the 1.2.8 release if I just apply immediately, definitely will give it a try for 1.3.0

Reply:
Something like this? @bgilbert 

```meson
dir = include_directories('.')
cc = meson.get_compiler('c')

test_code = ['mock.c', 'test.c', 'mark.c', 'sanity.c', 'common.c']

fossil_test_lib = library('pizza_test',
    test_code,
    install: true,
    include_directories: dir,
    dependencies: [cc.find_library('m', required: false)])

fossil_test_dep = declare_dependency(
    link_with: fossil_test_lib,
    include_directories: dir)

meson.override_dependency('fossil-test', fossil_test_dep)
meson.override_dependency('pizza-test', fossil_test_dep)
```

Reply:
Yup, exactly.